### DISCUSSION TITLE
How to get /snippets to work

### DISCUSSION THREAD

**Discussion by resistorsoftware** [saghen/blink.cmp#781] [Q&A]
Latest Neovim, blink and setup via LazyVim. I created a ~/.config/nvim/snippets directory and copied the rails.json from the friendly-snippets plugin extras. When I open a ruby file and try out the prefix "ba" (the first in the provided snippets) I do not get any response about before_action, but instead just get ruby snippet options. I am curious as to what else I need to do to enable these extra snippets. The instructions here [snippets](https://cmp.saghen.dev/configuration/snippets.html) reads to me like I can just copy the JSON into that created directory and it would work, as blink looks in that directory for any snippets by default. 

Can anyone help me get it to work and inform me of the folly of my ways?

**Comment by ddogfoodd** [comment]
Solved?

**Reply by resistorsoftware** [reply]
Yes. The instructions failed to mention one need also add a **_package.json_** file to the directory and add the snippet file as a directive in that directory. I picked up that from another plugin doing snippets and sure enough, when added, bingo. Snippets were there.
